src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -183,7 +184,7 @@ export const Deposit: React.FC<DepositProps> = ({
     accountId: accountId ?? null,
     amountCryptoBaseUnit: toBaseUnit(inputValues?.cryptoAmount, asset?.precision ?? 0),
     memo: thorchainSaversDepositQuote?.memo ?? null,
-    fromAddress,
+    fromAddress: fromAddress ?? null,
```

Comment:
should also be `T | undefined` in `useSendThorTx` but didn't feel like polluting this diff for this